packages/web/docs/src/content/product-updates/index.mdx

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
This isn't a new file, it was `packages/web/docs/src/pages/product-updates.mdx` before.